drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-kogito-client/src/test/java/org/drools/workbench/screens/scenariosimulation/kogito/client/converters/scesim/JSInteropApiConverterTest.java

```diff
@@ -0,0 +1,5 @@
+package org.drools.workbench.screens.scenariosimulation.kogito.client.converters.scesim;
+
+public class JSInteropApiConverterTest {
```

Comment:
Same comment of `ApiJSInteropConverterTest`

Reply:
@danielezonca For the same reason of `ApiJSInteropConverterTest`, I can test only a subset of the converter methods. For example, with `convertBackground` method, there are static JSInterops calls (eg. `JsUtils.toList(JsUtils.getUnwrappedElementsArray(getNativeFactMappingValue())`); which can't be mocked. For the other converter calls I used the approach you suggested, it seems really nice :)